tests/codegen/pow_of_two.rs

```diff
@@ -0,0 +1,68 @@
+// #[cfg(bootstrap)]
+// ignore-stage1
```

Comment:
I'm fairly sure we run these tests in CI, maybe there's a debug/release difference in this test. There are other codegen tests that explicitly set `-O` to avoid that, maybe that needs to be done here so that you can successfully run the test locally

Reply:
Changing the opt level in `// compile-flags:` didn't fix anything on its own.

Reply:
In the bors output:

> Skipping Suite(test::tests/codegen) because it is excluded

https://github.com/rust-lang/rust/actions/runs/7544739609/job/20538757975?pr=119911#step:26:1997
On line 1973

Reply:
I found at least part of the problem. The test had an error never should have passed. @oli-obk Do you know how how to override the test suite exclusion? The dev guide said to edit `[src/ci/github-actions/ci.yml](https://github.com/rust-lang/rust/blob/master/src/ci/github-actions/ci.yml)` but I don't know what to change to get it to run.

Reply:
Unfortunately I do not, but if the test is actually broken, you could fix it locally and then it will get tested when bors tries to merge it.

Reply:
Another part of the problem: The test was made before I refactored the pow function, so naturally it generates very different IR. I'll fix the last three functions once I finalize pow. Maybe I should make the test less precise. I won't mark this as resolved until after I fix it.

Reply:
I made the test easier to rewrite by making simplifying the LLVM IR. The `unchecked_*` functions will make it less volatile. In theory, it may also slightly encourage inlineing and increase debug compile and run times. Anyway, the test passes, and is no longer ignored by default with `--stage 1`